source/2_universal_payload.rst

```diff
@@ -2834,13 +2928,90 @@ The below tables show what nodes should be passed from bootloader and its corres
++======================+======================+=========================+=================================================================================================+
+| cpu-info             | memoryspace          | #number                 | Identifies the maximum physical memory addressability of the processor                          |
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+| Node                 | Property             | Value                   | Definition                                                                                      |
++======================+======================+=========================+=================================================================================================+
+| acpi                 | rsdp                 | <base address>          | Point to the ACPI RSDP table. The ACPI table need follow ACPI specification version 2.0 or above|
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+| Node                 | Property             | Value                   | Definition                                                                                      |
++======================+======================+=========================+=================================================================================================+
+| smbios               | entry                | <base address>          | Point to smbios structure UNIVERSAL_PAYLOAD_SMBIOS_TABLE                                        |
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+
++----------------------+----------------------+-------------------------+-------------------------------------------------------------------------------------------------+
+| Node                 | Property             | Value                   | Definition                                                                                      |
++======================+======================+=========================+=================================================================================================+
+| PayloadBase          | entry                | <base address>          | Point to ELF image base on permenant memory                                                     |
```

Comment:
on permanent memory or flash.

Reply:
I assume ELF on flash is compressed hence not able to be parsed?

Reply:
yes, I would like to support both cases.
if ELF is compressed, this base address will be permanent memory.
if ELF is not compressed, this base address will be flash.